### DISCUSSION TITLE
How to Use Ignored Devices Section

### DISCUSSION THREAD

**Discussion by wesbro** [node-alarm-dot-com/homebridge-node-alarm-dot-com#120] [Q&A] [answered]
How do I input devices to ignore? If I want to ignore a glass break sensor, and the ID for that sensor is 123456789-31, how would I input this I the field?

**Comment by ngori** [comment, marked as answer]
If you are using homebridge-config-ui-x you can add the ignored devices in the settings for homebridge-node-alarm-dot-com.   If not using the config-ui-x you can add them directly to your json as "ignoredDevices" with the the array of device ids.  It would look something like
  "ignoredDevices": [
                "123456789-31",
                "123456789-32"],

**Comment by chase9** [comment]
I just wrote a wiki page to help people with this in the future: https://github.com/node-alarm-dot-com/homebridge-node-alarm-dot-com/wiki/Ignoring-Devices#editing-the-config-manually
